<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RailArt</class>
 <widget class="QMainWindow" name="RailArt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>279</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RailArt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAchievementSystem">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>成就系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTicketGenerator">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>车票生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrainSearch">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>车次查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTripManager">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>行程管理</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="MainRailArt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
